pkg/volume/aws_ebs/aws_ebs.go

```diff
@@ -246,6 +246,10 @@ func (b *awsElasticBlockStoreBuilder) IsReadOnly() bool {
 	return b.readOnly
 }
 
+func (b *awsElasticBlockStoreBuilder) SupportsSELinux() bool {
+	return !b.IsReadOnly()
```

Comment:
I think I agree with @thockin, they're functionally equivalent but @thockin's way is less code.